public/locales/en/common.json

```diff
@@ -1,4 +1,38 @@
 {
-  "ONE": "One",
-  "TWO": "Two"
+  "nav": {
+    "home": "Home",
+    "about": "About",
+    "login": "Login",
+    "register": "Register"
+  },
+  "values": {
+    "voluntary": {
+      "heading": "Voluntary and pro-bono",
+      "content": "We are building a donation platform for the benefit of those in need and the organizations that help them, without seeking financial or other benefits from it, apart from satisfaction with the implemented campaigns and the support of those in need."
+    },
+    "proactive": {
+      "heading": "Proactive approach",
+      "content": "You don't wait to be given a task, you take it directly. This, of course, is true by correcting the structure a bit, so that at the moment it is not very clear what exactly the tasks are. However, more experienced people can easily see exactly what is missing and generate tasks that make sense. Take action, no wonder."
```

Comment:
"Take action, do not wonder" might be better

Reply:
```suggestion
      "content": "You don't wait to be given a task, you take it directly. This, of course, is true by correcting the structure a bit, so that at the moment it is not very clear what exactly the tasks are. However, more experienced people can easily see exactly what is missing and generate tasks that make sense. Take action, do not wonder."
```